pkg/cloud/services/networking/router.go

```diff
@@ -70,7 +70,7 @@ func (s *Service) ReconcileRouter(openStackCluster *infrav1.OpenStackCluster, cl
 		}
 		router = *createdRouter
 	} else {
-		s.scope.Logger().V(6).Info(fmt.Sprintf("Reuse existing Router %s with id %s", router.Name, router.ID))
+		s.scope.Logger().V(6).Info("Reuse existing Router", "name", router.Name, "id", router.ID)
```

Comment:
nit: `Reusing`? Also, `Router` is title-case here but lower-case above and elsewhere. We probably want the latter unless [@mdbooth decides otherwise](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1617#discussion_r1271990511).